commands/utils/packagehandlers/gopackagehandler.go

```diff
@@ -0,0 +1,15 @@
+package packagehandlers
+
+import (
+	"github.com/jfrog/frogbot/commands/utils"
+)
+
+type GoPackageHandler struct {
+	GenericPackageHandler
+}
+
+func (golang *GoPackageHandler) UpdateImpactedPackage(impactedPackage string, fixVersionInfo *utils.FixVersionInfo, extraArgs ...string) (shouldFix bool, err error) {
+	// In Golang, we can address every package as direct because of the way that 'go get' works
+	fixVersionInfo.DirectDependency = true
```

Comment:
Let's brainstorm to see how we can avoid the abuse of "DirectDependency" for indirect dependencies.